cmd/get/execution_util.go

```diff
@@ -6,25 +6,25 @@ import (
 	"io/ioutil"
 	"os"
 
+	"gopkg.in/yaml.v3"
+
 	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
 	"github.com/flyteorg/flyteidl/clients/go/coreutils"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
-
-	"sigs.k8s.io/yaml"
```

Comment:
For consistency we should probably convert other cli structs to also use the new gopkg.
Can you create an issue for it .